src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -345,7 +351,10 @@ function ReportActionItemMessageEdit(props) {
                     <View style={styles.editChatItemEmojiWrapper}>
                         <EmojiPickerButton
                             isDisabled={props.shouldDisableEmojiPicker}
-                            onModalHide={() => InteractionManager.runAfterInteractions(() => textInputRef.current.focus())}
+                            onModalHide={() => {
+                                setIsFocused(true);
```

Comment:
What's this line for? This is triggered on onFocus callback of Composer, isn't it?

Reply:
Yes, it is triggered in the `onFocus` callback. 

This line addresses an issue with Android, where the cursor was put in the beginning (video below), instead of at the correct position.

<details>
<summary>Video of the bug</summary>

https://github.com/Expensify/App/assets/104823336/09fcb4fe-cae0-404c-a5a0-72db1e9ea50f

</details>

If needed, I can investigate more this behavior and find another solution for that.

Reply:
I am thinking of creating platform files for focusWithDelay.js. So no need to delay with setTimeout on platforms that don't have this issue. What do you think?

Reply:
It's a great idea! 
I'm going to check which platforms need that delay, and which don't. And then create separate files for platforms.

Reply:
I found opposite behavior no matter this line is added or not:

Cursor is at the beginning at first but after picking emoji, it's inserted at the end


https://github.com/Expensify/App/assets/108292595/6c7358c9-31e9-417e-adef-81dacd33bab0

Reply:
Seems like the root cause is not related to this PR.
So when auto-focus first time, cursor position is at the beginning visually but at the end programatically.

Reply:
I confirmed that this also happens on main so it's separate issue